src/GrpcProtos/capture.proto

```diff
@@ -433,6 +433,15 @@ message FullCallstackSample {
   uint64 timestamp_ns = 4;
 }
 
+message TracepointCallstack {
+  // This is an internal (only used on the service side and won't be seen by the
+  // client) proto. This will be used to merge callstacks with tracepoints
```

Comment:
The first sentence can be simplified to "This proto is only used on the service and won't be seen by the client." without the need to use and define "internal".